src/olympia/browse/templates/browse/personas/category_landing.html

```diff
@@ -9,7 +9,11 @@
 } %}
 
 {% macro sort_link(key) -%}
-  {{ url('browse.personas', category.slug)|urlparams(sort=key) }}
+{% if category %}
+    {{ url('browse.personas', category.slug)|urlparams(sort=key) }}
+{% else %}
+    {{ url('browse.personas')|urlparams(sort=key) }}
+{% endif %}
```

Comment:
@diox is your "temporary fix" good enough to be committed?

Reply:
I'd prefer if this went separately, in another PR, with a matching test.

Reply:
I can write a test and submit another PR.